runtime/common/src/integration_tests.rs

```diff
@@ -288,12 +288,10 @@ fn run_to_block(n: u32) {
 	assert!(System::block_number() < n);
 	while System::block_number() < n {
 		let block_number = System::block_number();
-		AllPallets::on_finalize(block_number);
-		System::on_finalize(block_number);
+		AllPalletsWithSystem::on_finalize(block_number);
 		System::set_block_number(block_number + 1);
-		System::on_initialize(block_number + 1);
```

Comment:
System::on_initialize is now inside AllPalletsWithSystem. And it is no-op so having it after is fine.